components/uniset/src/uniset.rs

```diff
@@ -124,25 +125,42 @@ impl UnicodeSet {
         &self.inv_list
     }
 
-    /// Yields an iterator going through the character set in the [`UnicodeSet`]
+    /// Yields an [`Iterator`] going through the character set in the [`UnicodeSet`]
     ///
     /// # Examples
     ///
     /// ```
     /// use icu::uniset::UnicodeSet;
     /// let example_list = vec![65, 68, 69, 70];
     /// let example = UnicodeSet::from_inversion_list(example_list).unwrap();
-    /// let mut example_iter = example.iter_chars();
-    /// assert_eq!(Some('A'), example_iter.next());
-    /// assert_eq!(Some('B'), example_iter.next());
-    /// assert_eq!(Some('C'), example_iter.next());
-    /// assert_eq!(Some('E'), example_iter.next());
-    /// assert_eq!(None, example_iter.next());
+    /// let mut ex_iter_chars = example.iter_chars();
+    /// assert_eq!(Some('A'), ex_iter_chars.next());
+    /// assert_eq!(Some('B'), ex_iter_chars.next());
+    /// assert_eq!(Some('C'), ex_iter_chars.next());
+    /// assert_eq!(Some('E'), ex_iter_chars.next());
+    /// assert_eq!(None, ex_iter_chars.next());
     /// ```
     pub fn iter_chars(&self) -> impl Iterator<Item = char> + '_ {
         self.inv_list.chunks(2).flat_map(|pair| (pair[0]..pair[1])).filter_map(char::from_u32)
     }
 
+    /// Yields an [`Iterator`] returning the ranges of the inversion list in the [`UnicodeSet`]
```

Comment:
1. Don't assume people reading this know what an inversion list is. To users, a UnicodeSet should be an opaque set of code points.
2. Do the ranges mean that the characters are included or excluded? Please clarify.